parlai/agents/seq2seq/seq2seq.py

```diff
@@ -421,8 +422,9 @@ def valid(obs):
                     y.extendleft(reversed(x))
 
             max_y_len = max(len(y) for y in parsed_y)
-            for y in parsed_y:
-                y += [self.NULL_IDX] * (max_y_len - len(y))
+            parsed_y = [y if len(y) == max_y_len else
```

Comment:
this is padding things? should be a utility or something?

Reply:
you're right--this is padding. could be a utility but I'd like to land this first at least.